HISTORY.md

```diff
@@ -417,7 +417,7 @@ Library improvements
     respectively perform predicate function negation and function composition. For example,
     `map(!iszero, (0, 1))` is now equivalent to `map(x -> !iszero(x), (0, 1))` and
     `map(uppercase ∘ hex, 250:255)` is now equivalent to
-    `map(x -> uppercase(hex(x)), 250:255)` ([#17155]).
+    `map(x -> uppercase(base(16, x)), 250:255)` ([#17155]).
```

Comment:
I think this file should be left alone --- no need to try to rewrite history :)

Reply:
Don't tell Howard Zinn